partner-chains-cli/src/prepare_configuration/prepare_chain_params.rs

```diff
@@ -254,22 +295,19 @@ mod tests {
 		});
 
 		let mock_context = MockIOContext::new()
+			.with_json_file("payment.vkey", test_vkey_file_json())
+			.with_file(RESOURCES_CONFIG_FILE_PATH, "{}")
 			.with_json_file(CHAIN_ID.config_file, initial_chain_config).with_expected_io(vec![
-			scenarios::show_intro(),
-			MockIO::file_read(GOVERNANCE_AUTHORITY.config_file),
-			MockIO::prompt(CARDANO_CLI.name, CARDANO_CLI.default, "cardano-cli"),
-			MockIO::file_write_json(
-				CARDANO_CLI.config_file,
-				serde_json::json!({"cardano_cli": "cardano-cli"}),
-			),
-			prompt_and_save_to_existing_file(CARDANO_PAYMENT_VERIFICATION_KEY_FILE, "payment.vkey"),
-			MockIO::run_command("cardano-cli address key-hash --payment-verification-key-file payment.vkey", "76da17b2e3371ab7ca88ce0500441149f03cc5091009f99c99c080d9"),
-			save_to_existing_file(GOVERNANCE_AUTHORITY,  "0x76da17b2e3371ab7ca88ce0500441149f03cc5091009f99c99c080d9"),
-			MockIO::eprint("Governance authority has been set to 0x76da17b2e3371ab7ca88ce0500441149f03cc5091009f99c99c080d9"),
-			MockIO::eprint(CHAIN_ID_PROMPT),
-			prompt_with_default_and_save_to_existing_file(CHAIN_ID, Some("1"),"2"),
-			silently_fill_legacy_chain_params(final_chain_config),
-		]);
+				scenarios::show_intro(),
+				MockIO::file_read(GOVERNANCE_AUTHORITY.config_file),
+				prompt_and_save_to_existing_file(CARDANO_PAYMENT_VERIFICATION_KEY_FILE, "payment.vkey"),
+				MockIO::file_read("payment.vkey"),
+				save_to_existing_file(GOVERNANCE_AUTHORITY, GOV_AUTH),
```

Comment:
Cosmetic: it's a pity. GOV_AUTH is just short for GOVERNANCE_AUTHORITY, so basically the same name, but one is filename and one is key hex.

Reply:
Aliased the import and renamed the constant, it's unambiguous now.